### ISSUE TITLE
Diamond refactoring / Curve dollar incentive

### ISSUE THREAD

**Issue by zgorizzo69** [pavlovcik/uad-ui#606]
adapt `CurveDollarIncentive.sol` and `CurveDollarIncentive.t.sol` to fit the refactoring

**Comment by 0x4007** [comment]
I wish there was more detail here for anybody to pick up this task!

**Comment by 0xColm** [comment]
Hi @pavlovcik 
I'm Malcolm.
Nice to meet you.
As a smart contract developer, I'm very interested in this project. I have some experiences in development decentralized exchange platform like uniswap. I would be appreciate if you could let me know the possibility of picking this task.
Thank you.

**Comment by 0x4007** [comment]
I think @zgorizzo69 would need to put more details in the specification before we can consider this a ready bounty.

**Comment by FibrinLab** [comment]
> I think @zgorizzo69 would need to put more details in the specification before we can consider this a ready bounty.

Those contracts are currently not in the diamond. I guess this bounty is to include them and adapt tests. What's the time duration for this?

**Comment by 0x4007** [comment]
I asked @zgorizzo69 to reply here.

**Comment by zgorizzo69** [comment]
yep what about 1 day ?

**Comment by FibrinLab** [comment]
Can I have this?

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

PR made

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

Awaiting PR review

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

Awaiting review

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

Potential blocker in another PR. Waiting for merge

**Comment by 0x4007** [comment]
> another PR

Can you link here?

**Comment by FibrinLab** [comment]
> > another PR
> 
> 
> 
> Can you link here?

This also increments the facet like the "bonding curve" PR. Trying to maintain the PR order

**Comment by 0x4007** [comment]
> > > another PR
> > 
> > 
> > Can you link here?
> 
> This also increments the facet like the "bonding curve" PR. Trying to maintain the PR order

I don't understand your comment

**Comment by FibrinLab** [comment]
> > > > another PR
> 
> > > 
> 
> > > 
> 
> > > Can you link here?
> 
> > 
> 
> > This also increments the facet like the "bonding curve" PR. Trying to maintain the PR order
> 
> 
> 
> I don't understand your comment

Sorry about that. I was referring to the PR on "bonding curve". This Dollar Curve PR also increments facet.

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

Updating PR

**Comment by FibrinLab** [comment]
> Do you have any updates @FibrinLab? If you would like to release the bounty back to the DevPool, please comment `/unassign`

Waiting for merge
